qlib/backtest/exchange.py

```diff
@@ -394,9 +394,8 @@ def deal_order(
         if trade_account is not None and position is not None:
             raise ValueError("trade_account and position can only choose one")
 
-        trade_price = self.get_deal_price(order.stock_id, order.start_time, order.end_time, order.direction)
         # NOTE: order will be changed in this function
-        trade_val, trade_cost = self._calc_trade_info_by_order(
+        trade_price, trade_val, trade_cost = self._calc_trade_info_by_order(
```

Comment:
Please change the  docstring of `_calc_trade_info_by_order`
![image](https://user-images.githubusercontent.com/465606/128470187-1008d25f-dec0-4d61-b19c-973bb5da78e6.png)